src/main/java/gui/DashboardWindow.java

```diff
@@ -79,7 +79,7 @@ public DashboardWindow() {
         } catch (Exception e) {
             System.err.println("Problem with UI Manager");
         }
-        factory = OntologyLogic.getFactory();
+        factory = OntologyLogic.getFactory(OntologyLogic.defaultPathToOntology);
```

Comment:
Fajnie by było może użyć też tego przy wczytywaniu z pliku w GUI? Bo dosyć uciążliwe jest wybieranie tego pliku za każdym razem. @kbieniasz

Reply:
Mogę tak zrobić, że domyślnie jak nie wybierze się pliku to nie krzyczy, żeby wybrać plik tylko bierze ten domyślny